pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -19,6 +19,11 @@ isExecutable() {
         | grep -q '^ *Type: *EXEC\>\|^ *INTERP\>'
 }
 
+getElfClass() {
+    LANG=C readelf -h "$1" 2> /dev/null \
+        | grep '^ *Class:' | tr -s ' ' | cut -d ' ' -f 3
+}
```

Comment:
I'd rather avoid adding another call to `readelf`, because we already have `getSoArch` (which we could probably rename to `getElfArch` if it's not used outside of here), which also is way more specific about the architecture (like, eg. if you're on ARM).